docs/architecture/adr/0017-watchOS-use-swift.md

```diff
@@ -0,0 +1,57 @@
+---
+adr: "0017"
+status: Done
+date: 2022-12-30
+tags: [clients, watchOS]
+---
+
+# 0017 - Use Swift to build watchOS app
+
+<AdrTable frontMatter={frontMatter}></AdrTable>
+
+## Context and Problem Statement
+
+In order to build a watchOS application bundled to our iOS application we need to consider which
+technology to use. The requirements would be to remain in the same tech stack if possible and to
+have a watchOS app that we can build and can be bundled into our iOS app so that they are delivered
+together (the purpose is not to build an independant watchOS app).
+
+## Considered Options
```

Comment:
This section is a bit hard to digest, one approach to make it more readable would be to convert it to a pro/cons list for each option, either similar to https://contributing.bitwarden.com/architecture/adr/observable-data-services#pros-and-cons-of-the-options or just directly in this list.